.github/workflows/pre-commit_dependencies_notifier.yml

```diff
@@ -2,8 +2,7 @@ name: Warning maintainers
 on:
   pull_request_target:
     paths:
-      - requirements.txt
-      - requirements-opts.txt
+      - pyproject.toml
```

Comment:
wouldn't this wrongly notify you when you change anything in pyproject.toml (can be something other than the version bump).

tbh I don't like this message anyway and vote to remove it.

Reply:
Yes, the number of false positives would likely increase.
I'd be okay with removing since we have the dependabot workflow in place that takes care of applying the update in most scenarios